src/swift/flwr/Sources/Flower/Common/Parameter.swift

```diff
@@ -0,0 +1,214 @@
+//
+//  Parameter.swift
+//  FlowerSDK
+//
+//  Created by Daniel Nugraha on 09.01.22.
+//
+
+import Foundation
+import NumPySupport
+import PythonSupport
+import PythonKit
+import CoreML
+import NIOCore
+import NIOPosix
+
+let appDirectory = FileManager.default.urls(for: .applicationSupportDirectory, in: .userDomainMask).first!
+
+@available(iOS 14.0, *)
+public class ParameterConverter {
+    private var np: PythonObject?
+    
+    /// The permanent location of the numpyArray.
+    private var numpyArrayUrl = appDirectory.appendingPathComponent("numpyArray.npy")
+    private var group: EventLoopGroup?
+    
+    public static let shared = ParameterConverter()
+    
+    private init() {
+        initGroup()
+    }
+    
+    private func initGroup() {
+        if group == nil {
+            print("Opening Python event loop group")
+            group = MultiThreadedEventLoopGroup(numberOfThreads: 1)
```

Comment:
What is this group used for? Could you elaborate in a comment? Why do we use `numberOfThreads: 1` and not more? What would be the impact?

Reply:
We need Python to convert numpy to CoreML MultiArray, and since under the hood Python is still single threaded, using one thread will ensure the swift application to not violate PyStateGILCheck.

Reply:
Sounds reasonable. Can you add that as a comment?